modules/crr/pages/set-up-crr.adoc

```diff
@@ -32,17 +32,65 @@ Make sure TigerGraph is *not* installed with a local loopback IP such as 127.0.0
 The following setup is needed in order to enable Cross Region Replication.
 
 
+=== Back up primary data
 
-=== Backup primary data
-
-Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to create a backup.
+Use GBAR to xref:backup-and-restore:index.adoc[create a backup] of the primary cluster.
+See xref:backup-and-restore:index.adoc[Backup and Restore] for how to create a backup.
 
 If you are setting up both the primary cluster and the DR cluster from scratch, you can skip Steps 1, 2, and 4 and only perform Step 3.
 
 === Restore on the DR cluster
 
 Copy the backup files from every node to every node on the new cluster.  xref:backup-and-restore:index.adoc#_restore_from_a_backup_archive[Restore the backup] of the primary cluster on the DR cluster. See xref:backup-and-restore:index.adoc[Backup and Restore] on how to restore a backup.
 
+=== Delete metadata records
+
+After the restoration, but before enabling CRR on the DR cluster, delete the oudated Kafka metadata records.
+
+Kafka provides a script called `kafka-delete-records.sh`.
+It takes a JSON configuration file to specify which records/messages on which partition, and until which offset, you want to delete.
+This script runs on the Kafka partition level, so it requires configuration through all Kafka partitions of the Metadata topic.
+
+[source.wrap, console]
+----
+# get into kafka bin directory in DR cluster
+
+$ cd $(gadmin config get System.AppRoot)/kafka/bin/
+
+# Add JAVA to your system path. JAVA is already provided by TigerGraph.
+
+$ JAVA_HOME=$(dirname `find $(gadmin config get System.AppRoot)/.syspre -name java -type f`)
+$ PATH=$PATH:$JAVA_HOME
+
+# run kafka-delete-records to delete outdated records during restore
+
+$ bash kafka-delete-records.sh \
+  --bootstrap-server $(gmyip):30002 \
+  --offset-json-file <path to delete_config.json>
+----
+
+Here is an example of a `delete_config.json` file.
+
+[source.wrap, json]
+----
+{
+  "partitions": [
+    {
+      "topic": "<crr metadata topic name>",
+      "partition": 0,
+      "offset": -1
+    }
+  ],
+  "version": 1
+}
+----
+
+`<crr metadata topic name>` is the topic name that CRR consumes.
+It contains a prefix determined by the gadmin config entry `System.CrossRegionReplication.TopicPrefix`.
+By default, this prefix is an empty string.
+
+Example: If System.CrossRegionReplication.TopicPrefix is set to Primary, then metadata topic used by CRR on DR cluster will be named Primary.Metadata. Therefore, users need to pay attention when filling the topic name in the JSON config.
```

Comment:
Wrap "System.CrossRegionReplication.TopicPrefix" into code font like `System.CrossRegionReplication.TopicPrefix`.
Same for `Primary`. 

I also would like to rephrase the example like below, maybe using a table?
> Example: 
> If `System.CrossRegionReplication.TopicPrefix` is empty string on the cluster, then `<crr metadata topic name>` is `Metadata`. 
> If `System.CrossRegionReplication.TopicPrefix` is `Primary` on the cluster, then `<crr metadata topic name>` is `Primary.Metadata`.